backend/src/main/kotlin/org/loculus/backend/controller/FilesController.kt

```diff
@@ -61,4 +65,24 @@ class FilesController(
         response.sendRedirect(presignedUrl)
         return ResponseEntity.status(HttpStatus.FOUND).build() // 302 Redirect
     }
+
+    @GetMapping("/{accession}/{version}/{fileField}/{fileName}")
+    fun getFileUrl2(
+        @HiddenParam authenticatedUser: AuthenticatedUser,
```

Comment:
I don't really know how the kotlin code works, but just to flag that we definitely need to make sure that this user can access this file at some point :) [not blocking for dev stuff, just important to keep track of very carefully before going to prod]